### ISSUE TITLE
AbstractTrees

### ISSUE THREAD

**Issue by mkborregaard** [richardreeve/Phylo.jl#21]
There's this new package which looks pretty fundamental: https://github.com/Keno/AbstractTrees.jl 
The name of the key types overlap.

**Comment by richardreeve** [comment]
Yes, interesting - thanks. I wonder why I never came across it before...
